### PR TITLE
Update passwords to more directly reference using a password manager

### PR DESCRIPTION
**Pull request by deviantintegral**
This is very much a first draft for our meeting next week. Others may have different takes on where this doc should go, so feel free to push edits here or file a competing pull request!

### CODE REVIEW

File: access/passwords.md [hunk 1]
```diff
@@ -10,12 +10,12 @@ This policy applies to all Lullabot employees and contractors.
 All employees and contractors are expected to create strong passwords for access to all Lullabot and client accounts.
 
 ### Explanation and Implementation
-The first layer of defense that we have for our online accounts is the PIN, passcode, or password. As such, it is extremely important to use good, unique passwords, and keep them well protected. A good password consists of a fully random string, the longer the better. Contrary to popular belief, the inclusion of numbers, characters, or mixed case does not matter nearly as much as the length of the password itself.
+The first layer of defense that we have for our online accounts is the PIN, passcode, or password. As such, it is extremely important to use good, unique passwords, and keep them well protected. A good password consists is hard for people to guess and machines to brute-force. Contrary to popular belief, the inclusion of numbers, characters, or mixed case does not matter nearly as much as the length of the password itself.
```
Comment (YesCT) [inline access/passwords.md:13]:
```suggestion
The first layer of defense that we have for our online accounts is the PIN, passcode, or password. As such, it is extremely important to use good, unique passwords, and keep them well protected. A good password is difficult for people to guess and machines to brute-force. Contrary to popular belief, the inclusion of numbers, characters, or mixed case does not matter nearly as much as the length of the password itself.
```

Comment (YesCT) [reply access/passwords.md:13]: "consists is" -> "is"
"hard" -> "difficult" ... personal preference. difficult is more accurate, and translates to similar words in other languages.

File: access/passwords.md [hunk 2]
```diff
@@ -10,12 +10,12 @@ This policy applies to all Lullabot employees and contractors.
...
 
-Also, because you are now using one of these convenient tools, and would not be able to remember your passwords if you wanted to anyway, you might as well make them all super-secure. The length of a password is its primary strength. The longer it is, the stronger it is. These days, most security experts suggest passwords of 12-16 characters, minimum. But what does it matter to you if you are using copy/paste anyway? Crank those suckers up to 32 characters and be safe for the next millennium.
+With the use of a password manager, you no longer need to memorize each password. Passwords will be filled automatically, or can be copied and pasted in. The length of a password is its primary strength. The longer it is, the stronger it is. While our policy states a minimum of 16 characters because some poorly designed systems do not allow for longer passwords, in most cases it is fine to go up to 20 or 30 characters long.
```
Comment (YesCT) [inline access/passwords.md:19]: Nice documentation of the reason for 16 chars. That will help us know when to re-evaluate that min.

**Review by YesCT — CHANGES_REQUESTED** [review]
Nice improvement. I left some thoughts. One grammar suggestion seems worth requesting changes. Others aren't blocking.

**Review by YesCT — APPROVED** [review]
Nice improvement.

**Comment by deviantintegral** [comment]
We also discussed this today and it's good to go. Thanks team!

**Pull request merged**